.github/workflows/ci.yml

```
@@ -195,3 +195,42 @@ jobs:
 
       - if: github.repository != 'rust-lang/crates.io'
         run: pnpm test-coverage
+
+  e2e-test:
+    name: E2E / Test
```

```suggestion
    name: Frontend / Test (playwright)
```

I don't think E2E is quite right for this since it implies that the tests are also testing our backend code, which is currently not the case (and I'm not sure we'd want to do that, for performance reasons)

---

Fair enough! Thanks, I'll take this change.